stable/jenkins/templates/config.yaml

```diff
@@ -78,9 +85,34 @@ data:
                   </envVars>
                 </org.csanchez.jenkins.plugins.kubernetes.ContainerTemplate>
               </containers>
-              <envVars/>
+              <envVars>
+{{- range $index, $var := .Values.Agent.envVars }}
+                <org.csanchez.jenkins.plugins.kubernetes.PodEnvVar>
+                  <key>{{ $var.name }}</key>
+                  <value>{{ $var.value }}</value>
+                </org.csanchez.jenkins.plugins.kubernetes.PodEnvVar>
+{{- end }}
+              </envVars>
+              <ports/>
+              <livenessProbe>
```

Comment:
Here are examples of new config params that arent documented or maybe even needed (for example can we just use the defaults).

Reply:
Agree. Will include in next commit.